data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -1,31 +1,16 @@
 #!/usr/bin/env bash
 set -e
 
-mkdir --parents /etc/kubernetes/{manifests,bootstrap-configs,bootstrap-manifests}
-
-if ! podman inspect {{.ReleaseImage}} &>/dev/null; then
-    echo "Pulling release image..."
-    podman pull --quiet {{.ReleaseImage}}
-fi
+. /usr/local/bin/release-image.sh
 
-# convert the release image pull spec to an "absolute" form if a digest is available - this is
-# safe to resolve after the actions above because podman will not pull the image once it is
-# locally available
-if ! release=$( podman inspect {{.ReleaseImage}} --format '{{"{{"}} index .RepoDigests 0 {{"}}"}}' ) || [[ -z "${release}" ]]; then
-	echo "Warning: Could not resolve release image to pull by digest" 2>&1
-	release="{{.ReleaseImage}}"
-fi
+mkdir --parents /etc/kubernetes/{manifests,bootstrap-configs,bootstrap-manifests}
```

Comment:
same as https://github.com/openshift/installer/pull/2234/files#r315922772

Reply:
Can you clarify? I just moved this existing below sourcing release-image.sh, as I prefer to "import libraries" at the very top.

Reply:
mean for https://github.com/openshift/installer/pull/2234/files/956c2939e4cce7fc73130483deac2a91c6405c76#diff-ce82c1d8a44f7dfc41dfc024085ccfeeR4